tests/test_geometry.py

```diff
@@ -493,6 +493,58 @@ def test_baran_tetrazole():
     assert atom_types == expected_types
 
 
+def test_aniline():
+    # although aniline is slightly pyramidal, we'd rather it
+    # be assigned planar for safety.
+    mol = Chem.AddHs(Chem.MolFromSmiles("c1ccccc1N"))
+    atom_types = geometry.classify_geometry(mol)
```

Comment:
nit: should the naming be updated from `atom_types` -> `atom_geometries` as above?

Reply:
yes - i will add a note to redo this - but this will be a huge diff so I won't include it here.